lib/VM/JSLib/Intl.cpp

```diff
@@ -287,10 +288,10 @@ constexpr OptionData kDTFOptions[] = {
     {u"second",
      platform_intl::Option::Kind::String,
      kTimeRequired | kTimeDefault},
+    {u"fractionalSecondDigits", platform_intl::Option::Kind::Number, 0},
```

Comment:
just curious: is this change needed?

Reply:
It is not needed .. but thought it would be nice to keep it alongside rest of the time fields.